config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
The default value for new apps is `:json` however the app currently has 'marshal' cookies. This allows existing cookies to seamlessly migrate to the new setup. Else people who are currently logged in will get an exception page on production. This should be changed to `:json` once cookies are migrated/probably in a couple weeks?

Reply:
Meh, there are only two people who login at the moment. Shouldn't be an issue.